### DISCUSSION TITLE
ComboBox drop down list is transparent?

### DISCUSSION THREAD

**Discussion by xhuan8** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3737] [General]
```
<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesign2.Defaults.xaml" />
```
With the style "MaterialDesignOutlinedComboBox", the drop down list of ComboBox has transparent background, which is not the same as demo.
How to fix this issue?
![Untitled](https://github.com/user-attachments/assets/2e93062d-e7c3-4276-b656-f7f3ab2e1dc7)

**Comment by corvinsz** [comment]
@xhuan8 since this is not a problem in the demo app, it's most likely not a problem with this library, but rather one on your side.
Can you check if you have followed the [Getting started](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit?tab=readme-ov-file#getting-started) correctly and/or provide more information about your project?

**Comment by xhuan8** [comment]
@corvinsz here is a test project
[MaterialDesignTest.zip](https://github.com/user-attachments/files/18054772/MaterialDesignTest.zip)
![Untitled2](https://github.com/user-attachments/assets/93e07fcb-1050-4065-8139-df8d8b9f798e)

**Comment by corvinsz** [comment]
@xhuan8 the project you provided is not set up correctly. Again, please make sure to follow the [Getting started](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit?tab=readme-ov-file#getting-started) guide.
Alternatively I created a repo [here](https://github.com/corvinsz/MDIX3737CBBackground) which is basically your project with MDIX implemented correctly.

**Comment by xhuan8** [comment]
thanks, didn't notice the theme configuration.
